app/config/config.yml

```diff
@@ -231,6 +232,20 @@ nelmio_security:
     external_redirects:
         whitelist:
             - google.com
+            
+    # Force ssl.
+    # forced_ssl: ~
+        # uncomment to whitelist certain URLs from having ssl forced.
+        # enabled: true
+        # whitelist:
+        #     - ^/unsecure/
+    # Recommended security setting for cookies when ssl is turned on.
+    # session:
+        # cookie_secure: true
+    # Forces Microsoft/Chrome XSS-Protection with its block mode.
```

Comment:
In reverting the forced-SSL changes, the `xss_protection` block should be the only difference left.  Currenlty the move of where the `NelmioSecurityBundle` is loaded, the `requires_channel` changes in `security.yml`, and the duplicate `forced_ssl` block are still in the PR.